repose-aggregator/build.gradle

```diff
@@ -3,8 +3,23 @@ import java.text.SimpleDateFormat
 
 apply plugin: 'org.asciidoctor.convert'
 
+dependencies {
+    asciidoctor 'org.asciidoctor:asciidoctorj-diagram'
+}
+
 asciidoctor {
-    attributes 'source-highlighter': 'coderay'
+    requires 'asciidoctor-diagram'
+    attributes 'source-highlighter': 'coderay',
+               'stylesdir': "${project.projectDir}/src/docs/asciidoc/_stylesheets",
+               'stylesheet': 'repose.css',
+               'imagesoutdir': "${project.buildDir}/asciidoc/html5/images",
```

Comment:
I see you already fixed it on the REP-4712 branch, but I'll put it out there that this line breaks the diagram images.